File: output/ChatGPT/avatar/Java/C++/atcoder_ABC151_A.cpp
#include <iostream>

class RemRing {
    long module;
public:
    RemRing(long module) {
        this->module = module;
    }

    long sum(long a, long b) {
        return (a + b + module) % module;
    }

    long sub(long a, long b) {
        return (a - b + module) % module;
    }

    long prod(long a, long b) {
        return (a * b) % module;
    }

    long div(long a, long b) {
        return (a * inv(b)) % module;
    }

    long inv(long a) {
        long b = this->module;
        long u = 1;
        long v = 0;

        while (b > 0) {
            long t = a / b;
            a -= t * b;
            u -= t * v;

            long z = a;
            a = b;
            b = z;

            z = u;
            u = v;
            v = z;
        }

        u %= this->module;
        if (u < 0) u += this->module;
        return u;
    }
};

int main() {
    std::string s;
    std::cin >> s;

    std::cout << static_cast<char>(s[0] + 1) << std::endl;

    return 0;
}

File: output/ChatGPT/avatar/Java/Python/atcoder_ABC151_A.py
import sys

class RemRing:
    def __init__(self, module):
        self.module = module

    def sum(self, a, b):
        return (a + b + self.module) % self.module

    def sub(self, a, b):
        return (a - b + self.module) % self.module

    def prod(self, a, b):
        return (a * b) % self.module

    def div(self, a, b):
        return (a * self.inv(b)) % self.module

    def inv(self, a):
        b = self.module
        u = 1
        v = 0
        while b > 0:
            t = a // b
            a -= t * b
            u -= t * v
            z = a
            a = b
            b = z
            z = u
            u = v
            v = z
        u %= self.module
        if u < 0:
            u += self.module
        return u

def main():
    s = sys.stdin.readline().rstrip()
    print(chr(ord(s[0]) + 1))

# Execute the main function
if __name__ == "__main__":
    main()